src/test/java/ButtonClickTest.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ButtonClickTest {
+    @Test (testName = "ButtonClickTest")
+    public void testButtonClick() throws InterruptedException {
+        ChromeOptions options = new ChromeOptions();
+        options.setHeadless(true);
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://demoqa.com/elements");
```

Comment:
This code should not be there but moved to base class. You can just copy base code from `Automation QA 2.0` repo